src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -214,10 +214,14 @@ const PriceTextInput = ({
       >
         {focused ? (
           <>
-            <Row display={asset.lastPrice ? 'flex' : 'none'} marginRight="8">
-              LAST: {formatEth(asset.lastPrice)} ETH
-            </Row>
-            <Row display={asset.floorPrice ? 'flex' : 'none'}>FLOOR: {formatEth(asset.floorPrice)} ETH</Row>
+            {!!asset.lastPrice && (
+              <Row display={asset.lastPrice ? 'flex' : 'none'} marginRight="8">
+                LAST: {formatEth(asset.lastPrice)} ETH
+              </Row>
+            )}
+            {!!asset.floorPrice && (
```

Comment:
this wlil not render when `floorPrice===0`, is that intentional?

Reply:
yes this is intentional